images/kiosk/Makefile

```diff
@@ -2,24 +2,31 @@ mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
 COPYFILES   := 
 include ../Makefile.inc
 
-CMD=hb_wrapper.sh launch.sh browser.sh -i
+CMD=hb_wrapper.sh launch.sh hb_preload.sh run.sh -vvvvdim
+# http://127.0.0.1:8080/
+# hb_kiosk.sh
+# https://ix.de
+# hb_kiosk.sh
 ARGS=
-APP_ID=browser_app
 
-#APP=kiosktest
+# APP=kiosktest
 # IMG  = ${U}/${APP}:${IMAGE_VERSION}
 # TAG=titan.h-its.org:5555/${IMG}
 # titan.h-its.org:5555/hits/moon:${IMAGE_VERSION:-latest}
 
-
+# --ipc=host
+# -v /dev/shm:/dev/shm:rw
+# --device /dev/dri/card1
 # --privileged
-RUNTERM=--rm -it -a stdin -a stdout -a stderr --ipc=host --net=host 
+RUNTERM=--rm -it -a stdin -a stdout -a stderr --net=host -v /tmp/:/tmp/:rw -e DISPLAY --device /dev/dri/card1 --device /dev/dri/renderD129 
```

Comment:
Are you sure that devices will always be named that way?

Reply:
no, not always. I will remove those devices - they are just to try to get GPU support while checking Kiosk-Browser

Reply:
Would it be more robust if you would mount *all* devices in `/dev/dri/`?